libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -28,7 +28,6 @@
 #import "SFRootViewManager.h"
 #import "SFSDKWebUtils.h"
 #import "SFManagedPreferences.h"
-// FIXME-REORG #import "SFSmartStore.h"
```

Comment:
That's unrelated.
There is still a FIXME-REORG left in this file. And we have a bug opened for it.